README.md

```diff
@@ -1,43 +1,80 @@
 # Review Service
 
-> Provide review retrieval service for SpaceWork.com item Workspace item page.
+This service is setup to run in Docker containers. To run locally, follow instructions below.
 
-## Related Projects
+## Instructions For Running Locally
+1. Install Docker on your local machine https://docs.docker.com/get-docker/
+2. Install Docker Compose should your machine require extra steps https://docs.docker.com/compose/install/
+3. Clone down repository: "git clone https://github.com/sdc-perlman/review-service.git"
+4. From the root directory of the repository in the terminal, run "docker-compose up --build app mongo"
 
-  - https://github.com/space-work/nearby-workspaces
-  - https://github.com/space-work/amenities-service
-  - https://github.com/space-work/contact-widget-service
-  - https://github.com/space-work/workspace-service
-  - https://github.com/space-work/workspace-description-service
-  - https://github.com/space-work/photos-service
-  - https://github.com/space-work/location-service
+```console
+docker-compose up -- build app mongo
+```
+
+By specifiying the service, we are choosing which image to create containers of. If we only ran "docker-compose up", docker will assume that we want containers to run for all images
```

Comment:
I'm a little confused at this part (maybe because I don't know much about Docker). What outside of the service (app) and database would docker install?

Reply:
In the docker-compose file, each service is named. I have app, mongo, pg, and pgadmin. If we just run "docker-compose up", it will assume I want containers to run for each service, but if we name the services in the terminal with the command, it was exclude unnamed services. So in this case, I would just be running the app and mongo, and ignore pg and pgadmin.